protos/merlin/transformer/spec/upi_autoload.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+package merlin.transformer;
+
+option go_package = "github.com/gojek/merlin/pkg/transformer/spec";
+
+message UPIAutoload {
+    repeated string tableNames = 1;
+    repeated string variableNames = 2;
```

Comment:
Do we need to declare the type of variable?

Reply:
the only purpose of this autoload is to register tables and variables, so the type of variable is not checked

Reply:
What if the variable is used inside expression? Will it fail expr compilation?

Reply:
it won't fail compilation as long it is already declared in autoload or variable specs